src/main/java/net/minestom/server/entity/attribute/AttributeInstance.java

```diff
@@ -114,6 +116,14 @@ public AttributeModifier removeModifier(@NotNull AttributeModifier modifier) {
         return removeModifier(modifier.id());
     }
 
+    /**
+     * Clears all modifiers on this instance.
+     */
+    public void clearModifiers() {
+        this.modifiers.clear();
```

Comment:
This should not clear `LivingEntity.SPRINTING_SPEED_MODIFIER`, that is almost always invalid behavior.